src/cljs/hatti/views/dataview.cljs

```diff
@@ -95,8 +98,8 @@
                        (if (and (= view :map) no-geodata?)
                          [:a {:class "inactive" :title "No geodata"}
                           (name view)]
-                         [:a {:href (str "#/" (name view))
-                              :class (view->cls view)} label]))]
+                         [:a {:on-click (click-fn #(activate-view! view))
+                              :href "#" :class (view->cls view)} label]))]
```

Comment:
How does this play with client side routing? I think the reason the previous implementation did `(str "#/" (name view))` was to play well with Secretary, though I could be wrong about that.